apps/admin-gui/src/app/shared/components/delete-owner-dialog/delete-owner-dialog.component.html

```diff
@@ -0,0 +1,40 @@
+<h1 mat-dialog-title>{{'DIALOGS.DELETE_OWNER.TITLE' | translate}}</h1>
+
+<mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+
+<div mat-dialog-content *ngIf="!loading">
+  <p>
+    {{'DIALOGS.DELETE_OWNER.DESCRIPTION' | translate}}
+  </p>
+
+  <div class="font-weight-bold">
+    {{'DIALOGS.DELETE_OWNER.ASK' | translate}}
+  </div>
+
+  <table mat-table [dataSource]="dataSource" class="w-100">
+    <ng-container matColumnDef="name">
+      <th mat-header-cell *matHeaderCellDef></th>
```

Comment:
You maybe forgot to paste the name of this column. If you wanted to let the header empty, just ignore this comment.